mlir/utils/performance/perfRunner.py

```diff
@@ -1212,15 +1166,16 @@ def benchmarkFusionKernels(test_dir, paths: Paths, arch, numCU, tuningDb: MaybeT
         # Find the best perf_config
         bestPerf =""
         if tuningDb:
-            if (arch, testVector) in tuningDb:
-                bestPerf = tuningDb[arch, testVector]
+            configStr = config.toCommandLine()
```

Comment:
Without the toCommandLine call, testVector lacks the "-m conv" and "-t 1" that tuningDb has in the keys for conv entries.  Then it won't find a perfconfig and gives up.